<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>沙子</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>红砖</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>水泥</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加气块</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Sand_Square">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Sand_Vehicle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Sand_Bag">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>方</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>袋</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Common_Brick">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>块</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>袋</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Cement">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Aerated">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>块</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>合计：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>计算并保存数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>元(RMB)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_SUM">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
